lib/libs/email/content/newSubmission/emailTemplates/Waiver1915bCMS.tsx

```diff
@@ -18,10 +18,11 @@ export const Waiver1915bCMSEmail = (props: {
     | (Events["ContractingAmendment"] & CommonEmailVariables);
 }) => {
   const variables = props.variables;
-  const previewText = `${variables.authority} ${variables.actionType} Submitted`;
+  const actionTypeWithWaiver = formatActionTypeWithWaiver(variables.actionType);
+  const previewText = `${variables.authority} ${actionTypeWithWaiver} Submitted`;
```

Comment:
I don't see the point of these variables. I think it'd be easier to read if you just handle the strings in the props directly.